spec/models/application_qualification_spec.rb

```diff
@@ -30,4 +30,20 @@
         .to raise_error ArgumentError, "'invalid_level' is not a valid level"
     end
   end
+
+  describe 'auditing', with_audited: true do
+    it 'creates audit entries' do
+      application_form = create :application_form
+      application_qualification = create :application_qualification, application_form: application_form
+      expect(application_qualification.audits.count).to eq 1
+      application_qualification.update!(subject: 'Rocket Surgery')
+      expect(application_qualification.audits.count).to eq 2
```

Comment:
May be a personal preference, but would this read better if it were:

```ruby
expect { application_qualification.update!(subject: 'Rocket Surgery' }
  .to change { application_qualification.audits.count }.from(1).to(2)
```

or even

```ruby
expect { application_qualification.update!(subject: 'Rocket Surgery' }
  .to change { application_qualification.audits.count }.by(1)
```